client/containers/Configuration/Configuration.jsx

```diff
@@ -69,11 +69,11 @@ class Configuration extends Component {
           </div>
           <div className="row user-tabs">
             <div className="col-xs-12">
-              <Tabs defaultActiveKey={this.state.activeTab} animation={false}>
+              <Tabs defaultActiveKey={this.state.activeTab} animation={false} id="configurationTabs">
                 <Tab eventKey={1} title="Filter Hook">
                   <p>The <strong>filter hook</strong> allows you to specify which records are shown to the current users when loading the list of users or searching. For example: <i>Only show users from my department</i>. This has to be defined using the lucene syntax.</p>
                   <Editor
-                    value={this.state.scripts.filter}
+                    value={String(this.state.scripts.filter)}
```

Comment:
Editor props.type validation
  value: PropTypes.string.isRequired,
Without String() type is function